### ISSUE TITLE
Error with `deis certs:remove`

### ISSUE THREAD

**Issue by jwaldrip** [deis/deis#3533]
Getting the following error when trying to remove a cert.

```
$ deis certs:remove '*.brandfolder.com'
Removing *.brandfolder.com... 405 METHOD NOT ALLOWED
Detail:
Method 'DELETE' not allowed.
```

**Comment by bacongobbler** [comment]
can confirm with the following test. We don't support wildcard certificates right now, but you should be able to delete them!

```
diff --git a/controller/api/tests/test_certificate.py b/controller/api/tests/test_certificate.py
index 18fa891..9f54985 100644
--- a/controller/api/tests/test_certificate.py
+++ b/controller/api/tests/test_certificate.py
@@ -110,3 +110,11 @@ thejiQz0ThCMBw7QMpVOiSvYAlQG0ATsRYwdTDqENIWKlerOLCSuxmbqe8XeDKhq
         url = '/v1/certs/autotest.example.com'
         response = self.client.delete(url, HTTP_AUTHORIZATION='token {}'.format(self.token))
         self.assertEqual(response.status_code, 204)
+        # deleting a wildcard cert should work too (even though they're unsupported right now)
+        # https://github.com/deis/deis/issues/3533
+        Certificate.objects.create(owner=self.user,
+                                   common_name='*.example.com',
+                                   certificate=self.autotest_example_com_cert)
+        url = '/v1/certs/*.example.com'
+        response = self.client.delete(url, HTTP_AUTHORIZATION='token {}'.format(self.token))
+        self.assertEqual(response.status_code, 204)
```

**Comment by mboersma** [comment]
I was just hacking up the same test to see, thanks @bacongobbler. Will fix.

**Comment by carmstrong** [comment]
@jwaldrip Thanks for the report. This will be fixed with Deis v1.5.2, which will ship today or tomorrow morning.
